tests/test_rocket.py

```diff
@@ -20,7 +20,7 @@ def test_rocket(mock_show):
         nozzleRadius=33 / 1000,
         throatRadius=11 / 1000,
         interpolationMethod="linear",
-        grainsCenterOfMassPosition=0.39796,
+        grainsCenterOfMassPosition=-0.85704,
```

Comment:
Sorry, why was this change needed?

Reply:
Oh, it wasnt needed. Every other motor definition on the test_rocket.py is also like this. Will fix it soon